chat.js

```diff
@@ -137,7 +137,7 @@ class PatternTester {
 	register(...elems) {
 		for (let elem of elems) {
 			this.elements.push(elem);
-			if (/^[^ \^\$\?\|\(\)\[\]]+ $/.test(elem)) {
+			if (/^[^ ^$?|()[\]]+ $/.test(elem)) {
```

Comment:
It's 2 cases in the entire codebase I don't think it's a big deal to just listen to eslint here.